modules/programs/less.nix

```
@@ -0,0 +1,29 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.programs.less;
+in {
+  meta.maintainers = [ maintainers.pamplemousse ];
+
+  options = {
+    programs.less = {
```

I would suggest adding an enable option and use that in the `mkIf` below.
```suggestion
    programs.less = {
      enable = mkEnableOption "less, opposite of more";

```